draft-vvv-webtransport-quic.md

```diff
@@ -127,7 +127,7 @@ WebTransport streams are provided by creating an individual unidirectional or
 bidirectional QUIC stream.  WebTransport datagrams are provided through the QUIC
 datagram extension [QUIC-DATAGRAM].
 
-# Connection Establishment
+# Connection Establishment  {#connection}
```

Comment:
nit: extra space before {

Reply:
Apparently I've not been consistent about those.  Changed to two spaces everywhere for consistency.